sdk/eventhub/azure-eventhub/tests/perfstress_tests/README.md

```diff
@@ -0,0 +1,58 @@
+# EventHub Performance Tests
+
+In order to run the performance tests, the `azure-devtools` package must be installed. This is done as part of the `dev_requirements`.
+Start be creating a new virtual environment for your perf tests. This will need to be a Python 3 environment, preferably >=3.7.
+
+### Setup for test resources
+
+These tests will run against a pre-configured EventHub. The following environment variable will need to be set for the tests to access the live resources:
+```
+AZURE_EVENTHUB_CONNECTION_STRING=<the connection string of an Event Hub.>
+AZURE_EVENTHUB_NAME=<the path of the specific Event Hub to connect to>
+```
+
+### Setup for perf test runs
+
+```cmd
+(env) ~/azure-eventhub> pip install -r dev_requirements.txt
+(env) ~/azure-eventhub> pip install .
+```
+
+## Test commands
+
+When `azure-devtools` is installed, you will have access to the `perfstress` command line tool, which will scan the current module for runable perf tests. Only a specific test can be run at a time (i.e. there is no "run all" feature).
+
+```cmd
+(env) ~/azure-eventhub> cd tests
+(env) ~/azure-eventhub/tests> perfstress
```

Comment:
i think it's better to give a complete example here including a file name 
or perhaps something  like
``` perfstress <ClassName> --<args>```

Reply:
Sample command line is put at the bottom of the readme:

```
(env) ~/azure-eventhub/tests> perfstress ReceiveEventBatchTest --parallel=2 --event-size=1024 --num-events=100 --duration=100
```